mycli/output_formatter.py

```diff
@@ -67,23 +67,19 @@ def align_decimals(data, headers, **_):
     """
     pointpos = len(data[0]) * [0]
     for row in data:
-        i = 0
-        for v in row:
+        for i, v in enumerate(row):
```

Comment:
@meeuw `enumerate()` is one of those gems that's awesome once you find it. I think I found out about it from an article about idiomatic Python, [like this](http://python.net/~goodger/projects/pycon/2007/idiomatic/handout.html).